.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+* @elastisys/goto-kubernetes
+
+/rook/ @elastisys/goto-block-storage
```

Comment:
Should the QA group be used for `/release`?

Reply:
I discussed this a bit with @robinAwallace and said that we will skip adding QA to release folder for now to keep it simple.
We might change this and other smaller parts later if we feel like that would be better.